Tests/UnitTests/Core/Visualization/EventDataView3DBase.hpp

```diff
@@ -248,8 +248,7 @@ static inline std::string testMultiTrajectory(IVisualization3D& helper) {
   Vector3D rMom(1_GeV, 0.025_GeV * gauss(generator),
                 0.025_GeV * gauss(generator));
 
-  SingleCurvilinearTrackParameters<ChargedPolicy> rStart(cov, rPos, rMom, 1.,
-                                                         42.);
+  CurvilinearParameters rStart(cov, rPos, rMom, 1., 42.);
```

Comment:
What is the idea with the `deprecated constructors` - shall we not drop them or is that not feasible?

Reply:
I want to drop them, but that would add another 500+ lines of changes in the unit tests. As mentioned in the description, that should happen in a second PR, ideally still before the `v1.0.0` tag.

Reply:
I can do that today.

Reply:
Answered above.